src/routes/+page.svelte

```diff
@@ -50,8 +50,10 @@
 	<div class="section">
 		<h1>Latest update</h1>
 		<p>
-			<a href="/schedule">Schedule is announced!!</a> Time for last second tweaks to your talks, test
-			your demos and get ready for the big day!!
+			Registration is now open,
+			<a href="https://www.eventbrite.co.uk/e/ddd-southwest-2024-registration-798955125287">
+				get your tickets here</a
+			> and see you on the 27th!
```

Comment:
I know I approved, but just spotted this, is this a rogue new-line?

Reply:
yeah - vs code threw one in for me too when I did mine - the autoformatting seems a little weird, but the page still worked fine

Reply:
Cool, in which case, ignore me 😄